### PR TITLE
Improve 'get started' doc.

### LINKED ISSUE #150: Doc fixes for "Get started"
- [ ] do not use `start:blockchain:client`
- [ ] Improve “how to use with colonyJS” section in colonyNetwork (also link to that section from colonyJs)
- [ ] Be consistent with the trufflepig docs
- [x] Merge with Quickstart (and remove quickstart)
- [x] Create a new token in the example. Like this maybe: https://gist.github.com/chmanie/85f1c5cdf5a9f29b47096d05dac32acf

This is the process which needs to be documented:

1. clone the colonyNetwork to a new location using the command: `git clone --recursive https://github.com/JoinColony/colonyNetwork.git`
2. Then in the `colonyNetwork` dir do git checkout `ce9811a9f0fca53d9ab417d5fc24bbcf29c351c8`
3. In `colonyNetwork` do `yarn`
4. Install `ganache-cli` and `trufflepig` globally
5. Run in `colonyNetwork`: `ganache-cli -d —gasLimit 7000000 —acctKeys ganache-accounts.json`
6. In `colonyNetwork` do: `node_modules/.bin/truffle migrate`
7. In the same dir do: `trufflepig —ganacheKeyFile ganache-accounts.json`
8. Create a new project dir and initialise it as a new npm project.
9. After that do in that project dir: `yarn add @colony/colony-js-client @colony/colony-js-adapter-ethers @colony/colony-js-contract-loader-http ethers`
10. Then paste the following code into an `example.js` file in the project dir: [example.js · GitHub](https://gist.github.com/chmanie/85f1c5cdf5a9f29b47096d05dac32acf)
11. Run `node example.js` in the project dir

### PR DESCRIPTION
**Pull request by gichiba**
Aligns the step-by-step explanation of 'get started' with the actions
that are performed by scripts written in hackathonStarter/

Will close #150

### CODE REVIEW

File: docs/_Docs_GetStarted.md [hunk 1]
```diff
@@ -1,39 +1,105 @@
...
+
+After the contracts have deployed, you can start up TrufflePig, pointing it to the same accounts that you created with `ganache-cli`:
+```
+$ trufflepig --ganacheKeyFile ganache-accounts.json
+```
+
+
+### Initialize your project
+Create a new directory for your project, and initialize it with npm
+TODO: clarify what steps this hash
```
Comment (chmanie) [inline docs/_Docs_GetStarted.md]: This is probably still tbd

File: docs/_Docs_GetStarted.md [hunk 2]
```diff
@@ -90,17 +162,21 @@ const provider = new providers.JsonRpcProvider('http://localhost:8545/');
...
 
+Save the file, and run with `$ node example.js` - You should see your new cool colony and token appear on your private blockchain!
 
-## Task Life-cycle
-Once the colony is configured and everything has been initialized, you can start the task workflow.
+
+## The Task Life-cycle
+The most useful tool within a colony is the `task` method. Tasks are used to coordinate work, track reputation, and ultimately the only way to get paid through a colony. See [tasks](/colonynetwork/docs-tasks/) for a complete description of what tasks are within a colony.
 
 ### Create
```
Comment (chmanie) [inline docs/_Docs_GetStarted.md]: We should probably add a link to a section which explains how to get the spec hash?

Comment (gichiba) [reply docs/_Docs_GetStarted.md]: I have adjusted to include a short explanation that the hash could be a reference to an IPFS file, but  that it doesn't *have* to be.

**Comment by gichiba** [comment]
Ok, so I think this is complete enough to merge, and to re-deploy the docs to get these changes out there. Some people are already starting to mess around with the quickstart, so I think it's important that we get this out there, and then add the IPFS/hackathonStarter content in a subsequent round of edits.

**Pull request merged**